apis/serving/common/common_types.go

```diff
@@ -138,14 +138,17 @@ type PredictorStatus struct {
 	GrpcEndpoint string `json:"grpcEndpoint"`
 
 	//TODO TBC whether or not these are exposed here
-
 	//	// How many copies of this predictor's models are currently loaded - NOT YET SUPPORTED
 	//	// +kubebuilder:default=0
 	//	LoadedCopies int `json:"loadedCopies"`
 	//	// How many copies of this predictor's models are currently loading - NOT YET SUPPORTED
 	//	// +kubebuilder:default=0
 	//	LoadingCopies int `json:"loadingCopies"`
 
+	// Total number copies of this predictor's models
```

Comment:
nit:
```suggestion
	// Total number of copies of this predictor's models
```

I'm still kind of on the fence about whether to include the failed ones or not. I guess like you suggested we should in any case keep this consistent with what's done for the knative case (i.e. if some number of the scaled pods are failing)